### ISSUE TITLE
Error on rerunning composer install in ReloadCommand

### ISSUE THREAD

**Issue by makkus183** [OXIDprojects/oxrun#62]
After rerunning `composer install `(first execution when a vendor directory does not exist is working) with i got this error:

```
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Nothing to install, update or remove
Package phpunit/php-token-stream is abandoned, you should avoid using it. No replacement was suggested.
Generating autoload files
composer/package-versions-deprecated: Generating version class...
composer/package-versions-deprecated: ...done generating version class
64 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
Generating OXID eShop unified namespace classes ... Done
oxid-esales/oxideshop-composer-plugin: Installing component oxid-esales/developer-tools
PHP Fatal error:  Uncaught Error: Call to undefined method Oxrun\Command\Module\ReloadCommand::getDefaultName() in /var/www/html/vendor/symfony/console/DependencyInjection/AddConsoleCommandPass.php:61
Stack trace:
#0 /var/www/html/vendor/symfony/dependency-injection/Compiler/Compiler.php(140): Symfony\Component\Console\DependencyInjection\AddConsoleCommandPass->process(Object(Symfony\Component\DependencyInjection\ContainerBuilder))
#1 /var/www/html/vendor/symfony/dependency-injection/ContainerBuilder.php(789): Symfony\Component\DependencyInjection\Compiler\Compiler->compile(Object(Symfony\Component\DependencyInjection\ContainerBuilder))
#2 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Internal/Container/ContainerFactory.php(74): Symfony\Component\DependencyInjection\ContainerBuilder->compile()
#3 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Internal/Container/ContainerFactory.php(65): OxidEsales\EshopCommunity\Internal\Container\ContainerFactory->getCompiledSymfonyContainer()
#4 /var/www/html/vendor/oxid-esales/ in /var/www/html/vendor/symfony/console/DependencyInjection/AddConsoleCommandPass.php on line 61
composer [install] failed, composer command failed: exit status 255. stderr=
```

extracted `composer.json`

```
    "require": {
        "oxid-esales/oxideshop-metapackage-pe": "v6.3.1",
        "oxid-esales/graphql-storefront": "^1.0",
        "oxidprojects/oxrun": "^5.0", (installed through composer.lock is 5.0.0)
       .....
```

According [to this discussion](https://stackoverflow.com/a/61655652/1960760) maybe there is an explicit command missing in the service description in [services.yaml](https://github.com/OXIDprojects/oxrun/blob/4a6485d5fe851816faac06199c707f8a48a7d43f/services.yaml#L43):

**Comment by TumTum** [comment]
The problem is you used the `composer.phar` and not the one from the oxid `./vendor/bin/composer`
`composer.phar` unfortunately uses the old symfony/console v2 and this did not include the getDefaultName().

according to OXID it is good to run composer twice: https://docs.oxid-esales.com/eshop/de/6.2/installation/update/standard-update.html#schritt-abhangigkeiten-aktualisieren

```shell
composer install --no-plugins --no-scripts -no-dev
./vendor/bin/composer install --no-dev
```

**Comment by TumTum** [comment]
Is published in with PR #63

**Comment by makkus183** [comment]
@TumTum thx a lot for explaining and handling this!
